plotly_resampler/aggregation/aggregators.py

```diff
@@ -346,8 +347,9 @@ def _aggregate(
             idxs = (np.arange(n_out) * group_size).astype(int)
         else:
             x_ = x
```

Comment:
Why new variable?

Reply:
Not necessary at all!

![image](https://user-images.githubusercontent.com/18898740/230075513-b8208ede-eb48-4293-a956-79600724d755.png)